layouts/index.redirects

```diff
@@ -1,19 +1,16 @@
-{{- $docsLatest := index site.Params.versions.docs 0 -}}
-{{- $scalers    := where site.RegularPages "Section" "scalers" }}
-/docs     /docs/{{ $docsLatest }}     301!
+{{ $docsLatest := index site.Params.versions.docs 0 -}}
+
+/docs     /docs/2.6     301!
```

Comment:
Shouldn't we use this?
```suggestion
/docs     /docs/{{ $docsLatest }}     301!
```
Because once we release 2.7 that's no longer going to show the latest

Reply:
Oh, yes, of course! I was so focused on reworking the other rules, I missed this. Thanks for catching it!